### ISSUE TITLE
Training time

### ISSUE THREAD

**Issue by luocmin** [zylo117/Yet-Another-EfficientDet-Pytorch#512]
With the same data and Settings, It took me 10 hours to run one epoch on a server, but 30 minutes to run one epoch on my computer, and the bigger the batchsize, the longer it took. Do you have such questions? What is the cause?

**Comment by zylo117** [comment]
It is not normal. And I guess it's out of memory so that it ran on swap instead of RAM which is super slow.

**Comment by luocmin** [comment]
> It is not normal. And I guess it's out of memory so that it ran on swap instead of RAM which is super slow.

How do I look at memory

**Comment by luocmin** [comment]
> > It is not normal. And I guess it's out of memory so that it ran on swap instead of RAM which is super slow.
> 
> lcm是我，内存够呀
![image](https://user-images.githubusercontent.com/57932842/94382368-f5650200-016e-11eb-8467-33871bf8f8fa.png)
![image](https://user-images.githubusercontent.com/57932842/94382386-1168a380-016f-11eb-90d1-d2591260baf2.png)

**Comment by luocmin** [comment]
> > > It is not normal. And I guess it's out of memory so that it ran on swap instead of RAM which is super slow.
> > 
> > 
> > lcm是我，内存够呀
> > ![image](https://user-images.githubusercontent.com/57932842/94382368-f5650200-016e-11eb-8467-33871bf8f8fa.png)
> > ![image](https://user-images.githubusercontent.com/57932842/94382386-1168a380-016f-11eb-90d1-d2591260baf2.png)

**Comment by zylo117** [comment]
No, you really should check why it has only 1.9G free on your server while 95G in buffer.
